sw/device/silicon_creator/manuf/extensions/example_personalize_ext.c

```diff
@@ -16,7 +16,7 @@ status_t personalize_extension_pre_cert_endorse(
 }
 
 status_t personalize_extension_post_cert_endorse(
-    ujson_t *uj, manuf_certs_t *endorsed_certs,
+    ujson_t *uj, manuf_certs_t *perso_blob_from_host,
```

Comment:
`manuf_certs_t` type will no longer exist after this PR; replace with `perso_blob_t`?